src/app/views/communities/view/overview/overview.ts

```diff
@@ -107,13 +106,8 @@ export default class RouteCommunitiesViewOverview extends BaseRouteComponent {
 	setPageTheme!: ThemeStore['setPageTheme'];
 
 	feed: ActivityFeedView | null = null;
-	knownMembers: User[] = [];
-	knownMemberCount = 0;
 	finishedLoading = false;
-	owner: User | null = null;
-	collaborators: User[] = [];
-	collaboratorCount = 0;
-	initialCollaboratorCount = 0;
+	sidebarData: CommunitySidebarData | null = null;
```

Comment:
This can be passed as a prop from the parent "view" component. This way we only need to fetch once at the top-level. The way you do it is define a prop here, and then pass the prop into the <router-view> tag in the "view" component file. Vue-router will actually pass that prop data through into this component.

Reply:
We will still fetch it, just not process it due to backwards comp, right?
Old clients will still need the data to be returned from the overview endpoint.

Reply:
But sure, I'll pass it down through the router view.